website/src/pages/extensions/index.tsx

```diff
@@ -0,0 +1,23 @@
+import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
+import TailWindThemeSelector from '@site/src/components/TailWindThemeSelector';
+import Layout from '@theme/Layout';
+
+function ExtensionListFromIframe(): JSX.Element {
+  return (
+    <iframe
+      title="Extensions Catalog"
+      src="https://registry.podman-desktop.io"
+      className="flex w-full min-h-screen h-fit"
```

Comment:
The problem is the `min-h-screen`, which is translated to `min-height: 100vh`.

But 100vh mean the height of the screen, if the iframe is taking more than the entire screen to display, it will have a scrollbar